swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SessionView.swift

```diff
@@ -115,7 +113,7 @@ struct SessionView: View {
       Text("Disconnecting...")
     case .reasserting:
       Text("No internet connection. Resources will be displayed when your internet connection resumes.")
-    case .invalid, .none:
```

Comment:
I don't understand where `.none` came from and where it went.

Reply:
Yeah something like that.